From 1751bacda1b705f60e6ded9862d21b5bda127482 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@hoyer.xyz>
Date: Mon, 23 Oct 2017 10:23:42 +0200
Subject: [PATCH] Merge pull request #296 from
 dracut-mailing-devs/20170303132146.4968-1-crrodriguez@opensuse.org

systemd 233 requires sha256, hmac and algif_hash modules
---
 modules.d/00systemd/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/00systemd/module-setup.sh b/modules.d/00systemd/module-setup.sh
index 23d39bbe..d022de65 100755
--- a/modules.d/00systemd/module-setup.sh
+++ b/modules.d/00systemd/module-setup.sh
@@ -18,7 +18,7 @@ depends() {
 }
 
 installkernel() {
-    hostonly='' instmods autofs4 ipv6
+    hostonly='' instmods autofs4 ipv6 algif_hash hmac sha256
     instmods -s efivarfs
 }
 

